autoInstaller.sh

```diff
@@ -297,12 +301,14 @@ if $installRPMS; then
 fi
 
 if $installgtm || $installYottaDB ; then
+    gtmInst=true
     cd GTM
     ./install.sh $installydbOptions
     ./createVistaInstance.sh -i $instance $createVistaInstanceOptions
 fi
 
 if $installcache; then
+    gtmInst=false
```

Comment:
you can check for the absence of $installgtm/$installYottaDB instead of creating this variable